.github/workflows/test_functional.yml

```diff
@@ -108,7 +108,7 @@ jobs:
         run: |
           # install system deps
           brew update
-          brew install bash coreutils gnu-sed
+          brew install bash coreutils gnu-sed grep
```

Comment:
Install GNU grep.

Reply:
I think this may not work out of the box: I think you'll want to use `ggrep`: https://xebia.com/blog/learning-grep/